src/battle/battle.cpp

```diff
@@ -0,0 +1,26 @@
+#include "battle.h"
+#include "../utils/utils.h"
+#include "../creature/creature.h"
+#include "../creature/main_character/main_character.h"
+#include <iostream>
+
+void Battle()
+{
+	std::cout << "Battle starts!" << std::endl;
+	Creature Goblin(10, 14, 35, 16, 14, 10, 8, 7, 12, "Goblin");
```

Comment:
goblin is a variable, so it should start with a lower-case 'g'

Reply:
done